### ISSUE TITLE
Add a salad icon

### ISSUE THREAD

**Issue by FANMixco** [Templarian/MaterialDesign#6441]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

We are living in more greenish times and salads are more common than ever. I couldn't find anyone related. Maybe something easier than this:

### Give us an example of what this icon might look like.

![image](https://user-images.githubusercontent.com/1666156/151808497-b34d2e23-85de-48ed-8787-94318fd61e1e.png)

**Comment by Templarian** [comment]
Yeah I added `food-variant` as our healthy version of the burger + soda food icon.

Might make this one a little more "icon" centric like combining a leaf + bowl. The small 24x24 might lose context trying to make leaves coming out of the bowl.
